ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml

```diff
@@ -38,6 +38,15 @@
     - ansible_architecture == "x86_64"
   tags: patch_update
 
+- name: Add additional repositories for Raspian Buster
+  apt-repository: repo={{ item }}
+  with_items:
+    - deb-src http://raspbian.raspberrypi.org/raspbian/ buster main contrib non-free rpi
```

Comment:
What's the `deb-src` line needed for in this case?

Reply:
That will allow installation of `gcc-4.8`. On a fresh install of Raspbian Buster, it's already in `/etc/apt/sources.list` but commented out. See https://raspberrypi.stackexchange.com/a/101115

Reply:
I'm slightly tempted to just leave it out and just build on the default on that system, but can you add a comment saying that they're needed for gcc/g++ 4.8 (Also `s/Raspian/Raspbian`) :-)

Reply:
Commit in :-)